File: C/C Programs/PalindromeOrNot.c
#include<stdio.h>
void main()
{
	int num, rev =0, rem, n;
	printf("enter num \n");
	scanf("%d", &num);
	n= num;
	while(num != 0)
	{
		rem = num % 10;
		rev = rev*10+ rem;
		num /= 10;
	}
	//printf("%d", rev);
	if(n == rev)
	{
		printf("Palindrome Num \n");
	}
	else
	{
		printf("Not Palindrome \n");
	}
}
	

File: Java/Day1Programs/src/PalindromeOrNot.java
//10:Check whether the number is palindrome or not?

import java.util.Scanner;
public class PalindromeOrNot {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int revNum=0, orginalNum, rem;
		System.out.println("enter the number \n");
		int num = sc.nextInt();
		orginalNum = num;
		while(num!=0) {
			rem = num%10;
			revNum = revNum *10 +rem;
			num /= 10;
		}

		if(orginalNum == revNum) {
			System.out.println("Number is Palindrom");
		}
		else {
			System.out.println("Number is Not Palindrom");	
		}
		
	}

}
